cmd/update-notifier.go

```diff
@@ -27,6 +27,58 @@ import (
 	"github.com/fatih/color"
 )
 
+// humanizeDuration converts time.Duration to human readable duration.
```

Comment:
We can perhaps remove humanize duration and just use https://github.com/dustin/go-humanize/blob/master/times.go